package-lock.json

```diff
@@ -8807,6 +8808,14 @@
         }
       }
     },
+    "rollup-plugin-svg": {
+      "version": "1.0.1",
+      "resolved": "http://znpm.in.zillow.net:31050/r/rollup-plugin-svg/_attachments/rollup-plugin-svg-1.0.1.tgz",
```

Comment:
sorry, you need to use `--registry https://registry.npmjs.org/` when calling npm install, people on the internet dont' have access to Zillow's private network 😅